test/pool.test.ts

```diff
@@ -522,6 +522,25 @@ describe("EthPool", function () {
     }
     await sender.expectSetReceiverReverts(receiver, 1, "Too many receivers");
   });
+
+  it("Allows withdrawal of all funds", async function () {
+    const [sender] = await getEthPoolUsers();
+    const withdrawAll = await sender.pool.WITHDRAW_ALL();
+    const amount = 1234;
+    await sender.topUp(0, amount);
+    const balanceBefore = await sender.pool.signer.getBalance();
+    await submit(
+      sender.pool.withdraw(withdrawAll, {gasPrice: 0}),
```

Comment:
I’d feel more comfortable with the test if we also included some receivers to cover a reduction in the balance. However, if this is ok for you we can also leave as it is.

Reply:
Fixed